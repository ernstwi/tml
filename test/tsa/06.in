init foo
foo -> bar
    guard { x - y == 5 & z < 7}
bar invar { x < 10 }
bar -> baz
    guard { x >= 5 & z - y < 7}
