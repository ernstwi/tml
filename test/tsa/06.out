{
    "actions": [],
    "clocks": [
        "x",
        "y",
        "z"
    ],
    "edges": [
        {
            "action": null,
            "from": "bar",
            "guard": "x>=5&z-y<7",
            "reset": null,
            "to": "baz"
        },
        {
            "action": null,
            "from": "foo",
            "guard": "x-y==5&z<7",
            "reset": null,
            "to": "bar"
        }
    ],
    "locations": [
        {
            "id": "bar",
            "initial": false,
            "invariant": "x<10"
        },
        {
            "id": "baz",
            "initial": false,
            "invariant": null
        },
        {
            "id": "foo",
            "initial": true,
            "invariant": null
        }
    ]
}
